cmd/get-main.go

```diff
@@ -47,29 +47,32 @@ USAGE:
 FLAGS:
   {{range .VisibleFlags}}{{.}}
   {{end}}
-ENVIRONMENT VARIABLES:
-  MC_ENCRYPT:      list of comma delimited prefixes
-  MC_ENCRYPT_KEY:  list of comma delimited prefix=secret values
 
 EXAMPLES:
-  1. Get an object from S3 storage to local file system 
-    {{.Prompt}} {{.HelpName}} ALIAS/BUCKET/object path-to/object 
+  1. Get an object from MinIO storage to local file system 
+    {{.Prompt}} {{.HelpName}} play/mybucket/object path-to/object 
+
+	2. Get an object from MinIO storage using encryption
```

Comment:
Some alignment issue here, remember you need to remove any empty spaces otherwise the help will look misaligned